app/views/organisations/index.html.erb

```diff
@@ -0,0 +1,34 @@
+<% content_for :title, "Departments, agencies and public bodies - GOV.UK" %>
+
+<h1>Departments, agencies and public bodies - GOV.UK</h1>
+<br>
+<h2>Ministerial departments</h2>
+<%= @content.select { |item| item[:department_type] == "Ministerial" }.count %>
+<ul>
+  <% @content.each do |item| %>
+    <% if item[:department_type] == "Ministerial" %>
+    <li class="app-c-taxon-list__item"><%= link_to item[:title], item[:base_path] %></li>
```

Comment:
You're using a component class here instead of using the component itself. We should never reuse component classes outside of a component. There's some docs here on how to call the taxon-list component: collections.dev.gov.uk/component-guide/taxon-list (when you're running Collections locally) 🙂

For reference, it might be something like: 

```
<%= render 'components/taxon-list', {
          items: @content
      } %>
```